modules/registry-configuring-storage-vsphere.adoc

```diff
@@ -22,19 +22,20 @@ registry to use storage.
 vSphere volumes do not support the `ReadWriteMany` access mode. You must use
 a different storage backend, such as `NFS`, to configure the registry storage.
 ====
++
 * Must have "100Gi" capacity.
 
 .Procedure
 
 . To configure your registry to use storage, change the `spec.storage.pvc` in the
 `configs.imageregistry/cluster` resource.
-
++
 . Verify you do not have a registry pod:
```

Comment:
s/pod/Pod ?
Elsewhere in the docs (such as Storage), and in the [Documentation guidelines](https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc), Pod is capitalized.